src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.html

```diff
@@ -1,60 +1,59 @@
-<div class="modal-header">
-  <button type="button"
-          class="close float-right"
-          aria-label="Close"
-          (click)="modalRef.hide()">
-    <span aria-hidden="true">&times;</span>
-  </button>
-</div>
-<div class="modal-body">
-  <h2>
-    <img src="assets/Ceph_Logo_Stacked_RGB_120411_fa_348x348.png"
+<div class="about-container">
+  <div class="modal-header">
+    <button type="button"
+            class="close float-right"
+            aria-label="Close"
+            (click)="modalRef.hide()">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+  <div class="modal-body">
+    <img src="assets/Ceph_Logo_Standard_RGB_Reversed_120411_fa.png"
          class="ceph-logo"
          alt="{{ projectConstants.organization }}">
-    {{ projectConstants.organization }}
-  </h2>
-  <h3>
-    <strong>{{ projectConstants.projectName }}</strong>
-  </h3>
-  <div class="product-versions">
-    <strong>Version</strong>
+    <h3>
+      <strong>{{ projectConstants.projectName }}</strong>
+    </h3>
+    <div class="product-versions">
+      <strong>Version</strong>
+      <br>
+      {{ versionNumber }}
+      {{ versionHash }}
+      <br>
+      {{ versionName }}
+    </div>
     <br>
-    {{ versionNumber }}
-    {{ versionHash }}
-    <br>
-    {{ versionName }}
+    <ul class="list-unstyled">
+      <li class="row">
+        <strong class="col-6 col-sm-4">Ceph Manager</strong>
+        <span class="col-4 col-sm-4">{{ hostAddr }}</span>
+      </li>
+      <li class="row">
+        <strong class="col-6 col-sm-4">User</strong>
+        <span class="col-4 col-sm-4">{{ modalVariables.user }}</span>
+      </li>
+      <li class="row">
+        <strong class="col-6 col-sm-4">User Role</strong>
+        <span class="col-4 col-sm-4">{{ modalVariables.role }}</span>
+      </li>
+      <li class="row">
+        <strong class="col-6 col-sm-4">Browser</strong>
+        <span class="col-4 col-sm-4">{{ modalVariables.browserName }}</span>
+      </li>
+      <li class="row">
+        <strong class="col-6 col-sm-4">Browser Version</strong>
+        <span class="col-4 col-sm-4">{{ modalVariables.browserVersion }}</span>
+      </li>
+      <li class="row">
+        <strong class="col-6 col-sm-4">Browser OS</strong>
+        <span class="col-4 col-sm-4">{{ modalVariables.browserOS }}</span>
+      </li>
+    </ul>
```

Comment:
I see the current layout with too much space between the list components (and a risk of right-most elements to overlap with background picture). Instead of adding CSS classes to all list items, why not using a [HTML5 Description List DL Tag](https://www.w3schools.com/tags/tag_dl.asp) for this (specially suited for key-value lists):

```html
<dl>
  <dd>Ceph Manager</dd>
    <dt>{{ hostAddr }}</dt>
  <dd>User</dd>
    <dt>{{ modalVariables.user }}</dt>
  <dd>
...
</dl>
```

And then you can easily 'style' left & right components:
```css
.about-container dd {
    font-weight: ...;
    margin-right: ...;
}
```

Reply:
Ok. I'll make the changes

Reply:
@epuertat  I found two ways to do this. 

1) Using bootstrap classes for structure and making dt elements bold.
 style
```css
 .about-container dt {
  font-weight:bold;
}
```
markup
```html
<dl class=row>
      <dt class="col-4">Ceph Manager</dt>
      <dd class="col-8">{{ hostAddr }}</dd>
</dl>
```

2) using plain dl , dt and dd tags with flex-box to position them.
style
```css
    .about-container dl {
      display: flex;
      flex-wrap: wrap;
    }
   .about-container dt {
      flex:30%;
      margin:10px 0;
      font-weight:bold;
    }
    .about-container dd {
      flex:60%;
      margin:10px 0;
    }
```
markup
```html
   <dl>
      <dt>Ceph Manager</dt>
      <dd>{{ hostAddr }}</dd>
  </dl>
```
Which approach is best ?

Reply:
What about this third (mixed) alternative: you style the HTML tags (as in option 2), but using Bootstrap helpers (as [SASS mixins](https://sass-lang.com/documentation/at-rules/mixin) or variables). You may find a few helpers [here, like `make-col`](https://martijncuppens.github.io/bootstrap-mixins-documentation/docs/4.1/mixins/#make-col).

```scss
   .about-container dt {
      @include make-col(4);
      font-weight: $font-weight-bold;
    }
```

Reply:
Hi @epuertat. I've made changes. Please check.